static/css/components/buttonCta.less

```
@@ -133,6 +133,22 @@ a.cta-btn {
   }
 }
 
+// Button inside carousel
+// stylelint-disable-next-line selector-max-specificity
+.carousel-section .cta-btn {
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  font-size: 14px;
+  text-align: center;
+
+  &.cta-btn--external {
+    padding-right: 25px;
+    .btn-label {
+      padding: 8px 10px;
+    }
```

Ahhh it's apparently line 147! Hopefully this'll do the trick!

```suggestion
.carousel-section .cta-btn {
  white-space: nowrap;
  text-overflow: ellipsis;
  font-size: 14px;
  text-align: center;

  &.cta-btn--external {
    padding-right: 25px;
    // stylelint-disable-next-line selector-max-specificity
    .btn-label {
      padding: 8px 10px;
    }
```